run_service.sh

```diff
@@ -375,6 +403,17 @@ echo ""
 echo "This script will assist you in setting up and running the Trader service (https://github.com/valory-xyz/trader)."
 echo ""
 
+# Check the command-line arguments
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --with-staking)
+            read -n 1 -s -r -p "WARNING: the flag '--with-staking' is deprecated. Instead, use the environment variable 'USE_STAKING=true' stored in '$env_file_path'. Press any key to continue..."
```

Comment:
We could simplify the message to avoid confusion, because if the .env file is not yet created, it will prompt when reading the store in https://github.com/valory-xyz/trader-quickstart/pull/99/files#diff-91c4da23ec8a4f27b22bcaaf93a53e4a21cfeeb6a6977e9a3fdd994701ba3b40R351